api/routers/organization-role.js

```diff
@@ -6,7 +6,10 @@ router.use(sessionAuth);
 router.use(csrfProtection);
 
 router.get('/organization-role', OrganizationRoleController.findAllForUser);
-router.delete('/organization-role', OrganizationRoleController.destroy);
+// for reasons that aren't totally clear, this method doesn't receive a request
+// body when running normally but the test requests do pass the body properly
+// router.delete('/organization-role', OrganizationRoleController.destroy);
+router.delete('/organization/:org_id/user/:user_id', OrganizationRoleController.destroy);
```

Comment:
Would it make sense to move this to the `api/routers/organization`?  Your call.

_* Note: I'm not sure if any of the `organization-role` routes make much sense but not worth refactoring everything at this moment._